includes/admin/reporting/tabs/class.llms.admin.reporting.tab.students.php

```diff
@@ -94,13 +94,20 @@ public function breadcrumbs() {
 	 *
 	 * @return   void
 	 * @since    3.2.0
+	 * @since    [version] Added a check to see if the given user exists before outputting the report template.
 	 * @version  3.2.0
 	 */
 	public function output() {
 
 		// Single student.
 		if ( isset( $_GET['student_id'] ) ) {
 
+			llms_get_student( $_GET['student_id'] );
```

Comment:
```suggestion
```
I think this is a typo :D

Reply:
Ack! Thank you for catching that @eri-trabiccolo.